webscripts/webscripts-70/src/main/java/com/github/dynamicextensionsalfresco/webscripts/AnnotationWebscriptResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.github.dynamicextensionsalfresco.webscripts;
+
+import org.springframework.extensions.webscripts.WebScriptResponse;
+
+public class AnnotationWebscriptResponse extends AbstractAnnotationWebscriptResponse {
+
+    public AnnotationWebscriptResponse(WebScriptResponse response) {
+        super(response);
+    }
+
+    @Override
+    public void reset(String preserveHeadersPattern) {
```

Comment:
Is this the only difference in the 5 <-> 7 version? 

In that case, can we not just add this method without the `@Override` annotation?

Reply:
It appears this is the only "real" change yes.

Since `@Override` is only used at compile-time to check if that method actually overrides a method in the base class/interface, it can indeed be removed.

Even when `@Override` is not present and you do override a method in the base class/interface, you get a java compile warning at most.

Reply:
![image](https://user-images.githubusercontent.com/1098331/124474041-55dd3b80-dda0-11eb-8b47-32297522dc29.png)

I think we need to start over with this PR :grimacing: